app/components/common/scinapseInput/scinapseFormikCheckbox.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { withStyles } from "../../../helpers/withStylesHelper";
```

Comment:
Please change the position of the local module import

Reply:
I don't know what's wrong. That is working. I think I'll need more details.